src/modules/fundingManager/bondingCurveFundingManager/BancorVirtualSupplyBondingCurveFundingManager.sol

```diff
@@ -111,26 +117,35 @@ contract BancorVirtualSupplyBondingCurveFundingManager is
     ) external override(Module) initializer {
         __Module_init(orchestrator_, metadata);
 
-        address _acceptedToken;
-        IssuanceToken memory issuanceToken;
+        IssuanceToken memory issuanceTokenData;
+        address tokenAdmin;
         BondingCurveProperties memory bondingCurveProperties;
+        address _acceptedToken;
 
-        (issuanceToken, bondingCurveProperties, _acceptedToken) = abi.decode(
-            configData, (IssuanceToken, BondingCurveProperties, address)
+        (issuanceTokenData, tokenAdmin, bondingCurveProperties, _acceptedToken)
+        = abi.decode(
+            configData,
+            (IssuanceToken, address, BondingCurveProperties, address)
         );
 
-        __ERC20_init(
-            string(abi.encodePacked(issuanceToken.name)),
-            string(abi.encodePacked(issuanceToken.symbol))
+        ERC20Issuance _issuanceToken = new ERC20Issuance();
+        _issuanceToken.init(
+            string(abi.encodePacked(issuanceTokenData.name)),
+            string(abi.encodePacked(issuanceTokenData.symbol)),
+            issuanceTokenData.decimals,
+            issuanceTokenData.maxSupply,
+            tokenAdmin
```

Comment:
We can do that too. I'll add an initialMinter parameter so stuff can be configured out of the box (we need to indicate the addresses explicitly because of factory/initializer/etc shenaningans)